esphome/components/micro_wake_word/__init__.py

```diff
@@ -167,10 +214,21 @@ def _process_http_source(config):
     if not isinstance(manifest_data, dict):
         raise cv.Invalid("Manifest file must contain a JSON object")
 
-    try:
-        MANIFEST_SCHEMA_V1(manifest_data)
-    except cv.Invalid as e:
-        raise cv.Invalid(f"Invalid manifest file: {e}") from e
+    if manifest_version := manifest_data["version"]:
```

Comment:
We should use the const here, and also change to `.get` in case it is not in the file as we dont want to raise an unhandled key error
```suggestion
    if manifest_version := manifest_data.get(KEY_VERSION):
```